src/api/targomoClient.ts

```diff
@@ -78,6 +78,14 @@ export class TargomoClient {
     this.quality = new QualityClient(this)
   }
 
+  updateServiceUrl(region: string) {
```

Comment:
i personally would prefer `setServiceUrl` as it just does that. "Update" sound like some async task (updating some data) is involved.